tf2/include/tf2/buffer_core.h

```diff
@@ -88,7 +89,7 @@ static constexpr Duration BUFFER_CORE_DEFAULT_CACHE_TIME = std::chrono::seconds(
  *
  * All function calls which pass frame ids can potentially throw the exception tf::LookupException
  */
-class BufferCore
+class BufferCore : public virtual BufferCoreInterface
```

Comment:
@jacobperron and I spoke about this offline, and I think we can remove this virtual inheritance as well as a few other places in this pr and the rviz one. He's going to try to remove them and I'll continue my review after that. 👍

Reply:
Done 4160ac7